jax/experimental/compilation_cache/compilation_cache.py

```diff
@@ -67,9 +69,13 @@ def get_cache_key(xla_computation, compile_options) -> str:
     """
     hash_obj = hashlib.sha256()
     hash_obj.update(xla_computation.as_serialized_hlo_module_proto())
+    logging.vlog(1, f"get_cache_key hash after serializing computation: {hash_obj.digest().hex()}")
```

Comment:
Use the following idiom:

```
if logging.vlog_is_on(1):
  logging.vlog(1, blah)
```

Python evaluates the arguments eagerly and you don't want to calculate digest and hex when vlogging is *not* enabled

This is a common misuage of vlog, read
https://github.com/abseil/abseil-py/blob/master/absl/logging/__init__.py#L53-L57